000-vip-init.php

```diff
@@ -16,8 +16,12 @@
  * good reason not to.
  */
 
+if ( ! defined( 'ABSPATH' ) ) {
+	exit();
+}
```

Comment:
Bonus: fix potential path disclosure vulnerability. If PHP is configured to display errors, direct access to the file will disclose its path. If PHP is configured properly, direct access will generate a fatal error because of the undefined `add_action` function.